lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -607,6 +608,28 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  // https://dom.spec.whatwg.org/#dom-document-createcdatasection
+  createCDATASection(data) {
+    if (data.indexOf("[[>") !== -1) {
```

Comment:
I believe you want:

```js
if (data.indexOf("]]>") !== -1) {
```

Also, you might want to see whether setting the `nodeValue` actually does this escaping. But IIRC, inserting `]]>` directly in an XML CDATA section's node may not end up properly escaped, per browser behavior. (The [spec](https://dom.spec.whatwg.org/#dom-node-nodevalue) currently would suggest nothing happens at all actually (which would be even more unexpected), but I wonder whether this is a relic of having removed CDATA sections previously.)